crates/runtime/src/dataconnector.rs

```diff
@@ -93,8 +93,28 @@ pub enum Error {
     },
 }
 
+#[derive(Debug, Snafu)]
+pub enum DataConnectorError {
+    #[snafu(display("Cannot connect to {dataconnector} data connector. {source}"))]
```

Comment:
```suggestion
    #[snafu(display("Cannot connect to {dataconnector}. {source}"))]
```

I think the `data connector` is redundant here.